hypershift-operator/metrics.go

```diff
@@ -485,6 +504,15 @@ func (m *hypershiftMetrics) observeNodePools(ctx context.Context, nodePools *hyp
 	return nil
 }
 
+func (m *hypershiftMetrics) observeOperator(ctx context.Context, hypPod corev1.Pod) error {
+	hypVersion := version.GetRevision()
```

Comment:
When running the operator with no pod (e.g locally as a binary), `hypOpPods.Items[0]` will be nil, and this will panic

Reply:
In my first iteration I was asking for the deployment, but this has not the sha256 which is (IMHO) most important than the image itself. Maybe I can conditionally show one thing or other. WDYT?

Maybe in the future we can publish the image with the SHA256 instead of the tag (on releases), it will be much better for image use in self managed environments.